src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -251,6 +252,10 @@ public DynamoView(DynamoViewModel dynamoViewModel)
             this.dynamoViewModel.RequestEnableShortcutBarItems += DynamoViewModel_RequestEnableShortcutBarItems;
             this.dynamoViewModel.RequestExportWorkSpaceAsImage += OnRequestExportWorkSpaceAsImage;
 
+            AddPythonEngineToMainMenu();
+            PythonEngineManager.Instance.AvailableEngines.CollectionChanged += OnPythonEngineListUpdated;
+            dynamoViewModel.Owner = this;
```

Comment:
Glad you added this, this is the code I left out from my last PR. I was concerned about the memory impact, but I guess it is needed for the message window

Reply:
Yes, it was needed for this PR and many other dialog boxes which were displayed out of place.

Reply:
Yup, all good then